Controller/CRUDController.php

```diff
@@ -296,7 +295,6 @@ public function batchActionDelete(ProxyQueryInterface $query)
      *
      * @return Response|RedirectResponse
      *
-     * @throws NotFoundHttpException If the object does not exist
```

Comment:
This is thrown by parent method, not this one.

I just auto-generated this php doc block with phpstorm.